server/processors-alerts-publisher/src/main/java/org/hawkular/apm/processor/alerts/FragmentCompletionTimeAlertsPublisherMDB.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.apm.processor.alerts;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.ejb.ActivationConfigProperty;
+import javax.ejb.MessageDriven;
+import javax.inject.Inject;
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.MessageListener;
+import javax.jms.TextMessage;
+
+import org.hawkular.apm.api.model.events.CompletionTime;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * @author Juraci Paixão Kröhling
+ */
+@MessageDriven(name = "FragmentCompletionTimes_Alerts", messageListenerInterface = MessageListener.class, activationConfig = {
+        @ActivationConfigProperty(propertyName = "destinationType", propertyValue = "javax.jms.Topic"),
+        @ActivationConfigProperty(propertyName = "destination", propertyValue = "FragmentCompletionTimes"),
+        @ActivationConfigProperty(propertyName = "subscriptionDurability", propertyValue = "Durable"),
+        @ActivationConfigProperty(propertyName = "clientID", propertyValue = FragmentCompletionTimeAlertsPublisherMDB.SUBSCRIBER),
+        @ActivationConfigProperty(propertyName = "subscriptionName", propertyValue = FragmentCompletionTimeAlertsPublisherMDB.SUBSCRIBER),
+        @ActivationConfigProperty(propertyName = "messageSelector", propertyValue = "subscriber IS NULL OR subscriber = '"+ FragmentCompletionTimeAlertsPublisherMDB.SUBSCRIBER+"'")
+})
+public class FragmentCompletionTimeAlertsPublisherMDB implements MessageListener {
+    static final String SUBSCRIBER = "FragmentCompletionTimeAlertsPublisher";
```

Comment:
From what I remember on the code, it would re-publish the message to the queue, which is not something we want on this publisher. In the end, I think the Hystrix brings some advantages here, as it can both do a retry and batching.